app/models/user.rb

```diff
@@ -364,8 +364,8 @@ def cached_followed_tag_names
   end
 
   # methods for Administrate field
-  def banned
-    has_role? :banned
+  def suspended
```

Comment:
Maybe we should name this more idiomatically?

```suggestion
  def suspended?
```